bin/review-rot

```diff
@@ -122,7 +123,7 @@ if __name__ == '__main__':
                         help='Pull requests duration in terms of y=years,'
                              'm=months, d=days, h=hours, min=minutes')
     parser.add_argument('-f', '--format',
-                        default='oneline',
+                        default='online',
```

Comment:
Did you intend to make this change? I can see from the choices it's probably still meant to be "oneline".